src/Acme.UI/Controllers/Business/AthletesController.cs

```diff
@@ -1,31 +1,69 @@
-﻿using Examine;
-using Lucene.Net.Search;
+﻿using System.Collections.Generic;
 using System.Linq;
 using System.Web.Mvc;
+using Umbraco.Web;
 using Umbraco.Web.Mvc;
-
+using Examine;
+using Umbraco.Examine;
 
 namespace Acme.UI.Controllers.Business
 {
     public class AthletesController : SurfaceController
     {
+        public ActionResult GetAthlete(Umbraco.Web.PublishedModels.Athlete athlete)
+        {
+            return View("~/Views/Athletes/Athlete.cshtml", athlete);
+        }
+
+
+        public ActionResult AthleteDetails(int athleteId)
+        {
+            var athlete = Umbraco.Content(athleteId);
+            return View("~/Views/Athletes/Athlete.cshtml", athlete);
+        }
+
         // GET: Athletes
         public ActionResult List(int? maxAthletesToDisplay)
         {
-            if (ExamineManager.Instance.TryGetIndex("ExternalIndex", out var index))
+            var logginPhysioNodeId = GetLoggedInPhysioNodeId();
+            var athletes = new List<Umbraco.Web.PublishedModels.Athlete>();
+            if (logginPhysioNodeId != null)
             {
-                var searcher = index.GetSearcher();
-                var results = searcher.CreateQuery().Field("__NodeTypeAlias", "athlete").And().Field("path", "1212".MultipleCharacterWildcard()).Execute();
-                if (results.Any())
-                {
-                    int a = 1;
-                }
-                else
+                if (ExamineManager.Instance.TryGetIndex("ExternalIndex", out var index))
                 {
+                    var searcher = index.GetSearcher();
+                    var results = searcher.CreateQuery("content").ParentId(logginPhysioNodeId.Value).Execute();
 
+                    if (results.Any())
+                    {
+                        foreach (var item in results)
+                        {
+                            if (item.Id != null)
+                            {
+                                var node = Umbraco.Content(item.Id) as Umbraco.Web.PublishedModels.Athlete;
```

Comment:
remove the namespace with the using

Reply:
Have a read of this article that describes the different between Document, Content and Node.
Once you have done so lets discuss which is more appropriate and why.

Reply:
Done